community/numlockx/build

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+
```

Comment:
Remove this blank line.

Reply:
yes, now apply change for blank line and for deps. 
> numlockx depends on `libX11` `libXtst` and `libXext`, so you may want to add those as well
they weren't mentioned on the project page, I was only thinking of libXtst as optional since I didn't install it. But it is a fair observation. I will add them!